### ISSUE TITLE
Change Cultural Setup in Rest of the World

### ISSUE THREAD

**Issue by LordPeter** [ck2plus/CK2Plus_Rewrite#21]
Split Iranian (~same tier and thus treatment as Indo-Aryan):

1. Western Iranian (Kurdish, Baloch, Daylamite, Persian, Luri)
2. Eastern Iranian (Saka, Khwarezmian, Sogdian, Afghan[Pashto])

- move Azerbaijian to Turkic because apparently it isn't an Iranian culture (my mistake). And maybe also rename them to "Azeri" because it sounds more... idk, _archaic_, and less like the modern country.

(All credit for ideas go to Wikipedia)

**Comment by LordPeter** [comment]
Split Arabic again into Maghrebi & regular Arabic (as was before).

**Comment by LordPeter** [comment]
Split Latin(Gallo-Romance):

1. Gallo-Romance (French, Norman, Arpitan)
2. Occitano-Romance (Occitan, Catalan)

- Catalan is moved to the new group from Iberian.
Reasoning: https://en.wikipedia.org/wiki/Occitano-Romance_languages
https://en.wikipedia.org/wiki/File:Romance-lg-classification-en.svg

But because they have somewhat cultural ties with their old groups, Catalan later gets a small opinion bonus towards Iberian, and Occitan a small opinion bonus towards Gallo-Romance.

**Comment by LordPeter** [comment]
Split Caucasian again, because these are all pretty isolated languages/cultures in the mountains:

1. Kartvelian (Georgian)
2. West Caucasian (Circassian)
3. North Caucasian (Caucasian Avar)

- Caucasian Avar would be a new culture based on the historical people who are unrelated to the current _Pannoinian_ Avars currently ingame: https://en.wikipedia.org/wiki/Avars_(Caucasus)
These are rather isolated culture group, but would get opinion boni towards each other for geographical proximity (so they'd form a "Caucasian supergroup" like Iranian, Indo-Aryan, Germanic etc will do)

**Comment by LordPeter** [comment]
- Add Veps/Vepsian to Finnic group

https://en.wikipedia.org/wiki/Vepsians

**Comment by daniloy** [comment]
So you want to split Arabic and Maghrebi too?

**Comment by daniloy** [comment]
> 
> 
> Split Latin(Gallo-Romance):
> 
>     1. Gallo-Romance (French, Norman, Arpitan)
> 
>     2. Occitano-Romance (Occitan, Catalan)
> 
> 
>     * Catalan is moved to the new group from Iberian.
>       Reasoning: https://en.wikipedia.org/wiki/Occitano-Romance_languages
>       (Quote: "The issue at debate is as political as it is linguistic because the division into Gallo-Romance and Ibero-Romance languages stems from the current nation states of France and Spain and so is based more on territorial criteria than historic and linguistic criteria.")
>       https://en.wikipedia.org/wiki/File:Romance-lg-classification-en.svg
> 
> 
> But because they have somewhat cultural ties with their old groups, Catalan later gets a small opinion bonus towards Iberian, and Occitan a small opinion bonus towards Gallo-Romance.

This would make the Occitan rebel against France. I would like to keep it that way for balance reasons.

**Comment by daniloy** [comment]
> 
> 
> Split Arabic again into Maghrebi & regular Arabic (as was before).

can do

> 
> 
>     * Add Veps/Vepsian to Finnic group
> 
> 
> https://en.wikipedia.org/wiki/Vepsians

ditto

**Comment by daniloy** [comment]
> 
> 
> Split Iranian (~same tier and thus treatment as Indo-Aryan):
> 
>     1. Western Iranian (Kurdish, Baloch, Daylamite, Persian, Luri)
> 
>     2. Eastern Iranian (Saka, Khwarezmian, Sogdian, Afghan[Pashto], Alan)
> 
> 
>     * move Azerbaijian to Turkic because apparently it isn't an Iranian culture (my mistake). And maybe also rename them to "Azeri" because it sounds more... idk, _archaic_, and less like the modern country.
> 
>     * move Alan back to Iranian (eastern), because Caucasian is to be split and they would then belong here (as before).
> 
> 
> (All credit for ideas go to Wikipedia)

I was thinking this myself. 
I left the Azeris there because of cultural reasons.

**Comment by daniloy** [comment]
> Split Caucasian again, because these are all pretty isolated languages/cultures in the mountains:
> 
>     1. Kartvelian (Georgian)
> 
>     2. West Caucasian (Circassian)
> 
>     3. North Caucasian (Caucasian Avar)
> 
> 
>     * Caucasian Avar would be a new culture based on the historical people who are unrelated to the current _Pannoinian_ Avars currently ingame: https://en.wikipedia.org/wiki/Avars_(Caucasus)
> 
> 
> These are rather isolated culture groups, but would get opinion boni towards each other for geographical proximity (so they'd form a "Caucasian supergroup" like Iranian, Indo-Aryan, Germanic etc will do)

I know they lingustically isolated and perhaps culturally, but we would have so many one-culture groups. Wouldnt it be feasible just to have one vague geographic grou?

**Comment by LordPeter** [comment]
Good point about Azeri. I just read a bit closer and indeed, the language change to Turkic only happened from the 11th century on, and "Turkic Azerbaijani" was only dominant from the 16th century on (https://en.wikipedia.org/wiki/Old_Azeri_language).
So you are right, it should stay in Iranian (Western).

**Comment by LordPeter** [comment]
About Occitans rebelling against France: That is right, it _would_ upset the previous balance.
But... isn't that historically authentic?
https://en.wikipedia.org/wiki/Duchy_of_Aquitaine#Early_history
https://en.wikipedia.org/wiki/Aquitaine#Ethnic_make-up_in_the_Early_Middle_Ages
It would seem that while they did still have rulers of the local culture, they tried at every chance to become independent from the Franks. And only once Frankish rulers were in charge the rebellions of the elite stopped (I can't find info about local population sentiment... but eh, who cares about peasants).
I agree that the issue at hand is a delicate one. But given that we have already decided to tread that path for all the other culture groups, I do not want to sacrifice consistency for balance reservations _in a single instance_.
Anyway, _if_ this new group should be made, it would need to give an opinion bonus towards Gallo-Roman so that they don't hate each other that much.
So it depends on whether that is actually possible... if it is, I would like it to be e.g. -10 between Gallo-Roman cultures (French vs Norman), -20 between Gallo-Romance and unrelated group (e.g. French vs Novgorodian), and an intermediate -15 between Gallo-Romance and Occitan (e.g. French vs Occitan). So that they would only be _slightly_ more rebellious.

**Comment by Whizzer** [comment]
I can agree with all changes, outside of splitting the Caucasian group into three, unless we add more cultures to them.

**Comment by daniloy** [comment]
> 
> 
> About Occitans rebelling against France: That is right, it _would_ upset the previous balance.
> But... isn't that historically authentic?
> https://en.wikipedia.org/wiki/Duchy_of_Aquitaine#Early_history
> https://en.wikipedia.org/wiki/Aquitaine#Ethnic_make-up_in_the_Early_Middle_Ages
> It would seem that while they did still have rulers of the local culture, they tried at every chance to become independent from the Franks. And only once Frankish rulers were in charge the rebellions of the elite stopped (I can't find info about local population sentiment... but eh, who cares about peasants).
> I agree that the issue at hand is a delicate one. But given that we have already decided to tread that path for all the other culture groups, I do not want to sacrifice consistency for balance reservations _in a single instance_.
> Anyway, _if_ this new group should be made, it would need to give an opinion bonus towards Gallo-Roman so that they don't hate each other that much.
> So it depends on whether that is actually possible... if it is, I would like it to be e.g. -10 between Gallo-Roman cultures (French vs Norman), -20 between Gallo-Romance and unrelated group (e.g. French vs Novgorodian), and an intermediate -15 between Gallo-Romance and Occitan (e.g. French vs Occitan). So that they would only be _slightly_ more rebellious.

Or we put Catalan into Gallo Romance

**Comment by daniloy** [comment]
> 
> 
> I can agree with all changes, outside of splitting the Caucasian group into three, unless we add more cultures to them.

I concur and I would leave Alan because it is a geographic group. Also, eastern Iranian groups are also very far away. I also think the centuries of isolation from the other groups would have rendered it quite different ( just a hypothesis)

**Comment by LordPeter** [comment]
About Caucasian: Sure, that would make a lot of single culture groups.
But my point is, it would be consistent. I just don't see why we would have single culture groups for Basque, Coptic, Assyrian, Hungarian & Tocharian, and then bundle together three cultures that are _linguistically_ less related even than Assyrian & Coptic.
If you look at the maps, the Caucasus region indeed is weird and unqiue in that it has so many unrelated groups so close together. https://upload.wikimedia.org/wikipedia/commons/b/b7/Primary_Human_Languages_Improved_Version.png
Idk why this is the case, but it is. And while gameplay is a valid concern, the region remains highly unstable to this day, so frequent rebellions would not seem implausible to me.
And reading on, there seems to have been significant differences also in regards to culture other than language: https://en.wikipedia.org/wiki/Circassians#Culture
Where Georgians where somewhat influenced by Christianity and Romans, Circassians seem to have had a unique social system, and the Northern group had a fascinating tower architecture https://en.wikipedia.org/wiki/Vainakh_tower_architecture.

(And now I prepared this big post and someone commented. So I will post it anyway...)

You would split it if we added more cultures? That would not be a problem.
I have a few more candidates; but now I'm getting wary because do we have enough provinces to assign them all? Longer term I would just add some more to the area, but until we go back to editing the map it might be problematic.

Anyway, e.g. this would be an additional candidate:
- https://en.wikipedia.org/wiki/Udi_people descendants of https://en.wikipedia.org/wiki/Caucasian_Albania (almost extinct today, unique in that it had its own script: https://en.wikipedia.org/wiki/Caucasian_Albanian_script)

**Comment by LordPeter** [comment]
"Or we put Catalan into Gallo Romance"
But that would defeat the whole purpose! My aim is to group cultures together that are well, _culturally related_, and dampen the resulting gameplay upset by introducing opinion boni. Trying to make an exception for Occitan just so that the current gameplay situation is preserved does not make sense to me. Then we should not have split Indo-Aryan to begin with - or German long before.

"I concur and I would leave Alan because it is a geographic group. Also, eastern Iranian groups are also very far away. I also think the centuries of isolation from the other groups would have rendered it quite different ( just a hypothesis)"
Okay. But why would it be a geographic group? **None** of the other groups are purely based on geography. I can see why it is tempting here, but it would not make sense if every region of the world is treated equally.
SWMH interestingly has taken exactly that approach, for every culture: They just randomly group Circassian with Altaic (!), Coptic with East African and so forth. Just because they are in the same area. It is a possible approach, but one we have not taken for anything yet, and one that I don't find particularly appealing or plausible.

...
...
I realize I might come to sound a lot like the Kaiser I never met and that left before I joined, with my newfound fervor about the issue. I apologize, it isn't my aim to upset you with my personal agenda.
In the end, I hope to convince you to my view, not impose it on you like a Kaiser.
I'm just getting all worked up, sorry, idk why ^^

**Comment by schwarherz** [comment]
Each of these really should be their own issue, just saying. One blanket issue for the whole world is kinda...broad

**Comment by daniloy** [comment]
@LordPeter 

I suggest the following: We use your group for Occitan/Catalan.

We do have cultural groups based on location: East/West/Central African. The Problem with the Caucasus is that yes it is a boatload of language groups. Do we have to give them all an lone group and think it is worth it, then do it but here is the twist: you are gonna add them :D.

I will do the rest.

**Comment by LordPeter** [comment]
@schwarherz you are right, I was too greedy. I will use the appropriate issues now.

@daniloy: Great, I hoped you'd agree! I can make the Caucasian split, no problem. I'm just waiting on someone to confirm my plans for opinion boni to be working; otherwise I might have to rethink my stance again.
As for Africa... haven't thought about it yet, I actually assumed it was non-geographic. If it turns out I have a complaint there as well, I'll post in the other issue.

**Comment by LordPeter** [comment]
Just a rather small one here, but a quite unique one imo that deserves to not be overlooked.

**Eastern Iranian**
- Nuristani (https://en.wikipedia.org/wiki/Nuristanis) _(technically would be a separate culture-group linguistically, but that would be stretching it)_

Legend:
"+" denotes existing Plus culture
"*" denotes suggested new culture
"?" denotes an undecided candidacy
"->" denotes a renaming suggestion
"[ ]" denotes an alternate name for clarification, which need not be changed to
"#future" denotes cultures planned for inclusion after the map transition, when more parts of the map are visible

**Comment by schwarherz** [comment]
@lordpeter just remember that language and culture are not the same thing

**Comment by LordPeter** [comment]
True. Though we have used it as a guidance now for which cultures to group in most places (as opposed to a purely geographical grouping as SWMH does, and a mixed approach which is inconsistent and has been officially discreated by me). So for technical reasons we now often equate language_group = culture_group, but not necessarily language = culture.
But this approach is useful only if the language groups in question are roughly equal in size, which isn't the case for Nuristani (https://en.wikipedia.org/wiki/Nuristani_languages). It is of negligible size compared to the greater East Iranian and West Iranian. And they are also somewhat related as opposed to the completely unrelated Caucasus groups. Which is why I would just put them with East Iranian here.

**Comment by LordPeter** [comment]
I'd actually also like to keep Nuristani. The Afghanistan region is quite tribal even today, and the Nuristanis have supposedly been around for a very longtime, even looking different from the surrounding population.

**Comment by questionmarkexclamationpoint** [comment]
@daniloy Is this one done?
